README.md

```diff
@@ -4,7 +4,26 @@
 [![Chat](https://img.shields.io/static/v1.svg?label=chat&message=mattermost&color=145dbf)](https://mattermost.eclipse.org/eclipse/channels/eclipse-codewind)
 
 # Codewind for VS Code
-Create and develop cloud-native, containerized web applications from VS Code.
+Codewind for VS Code provides tools to help build cloud-native, containerized applications from VS Code, regardless of which runtime or language you use. Rapidly create an application from a template project and launch, update, test, and debug in Docker containers on the desktop.
+
+## Why use Codewind?
+- Get started quickly with templates or samples.
+- Use the tools, runtimes, and languages that you're already familiar with.
+- Pull in existing applications and get them ready for the cloud.
```

Comment:
"Pull in your existing applications, and let Codewind help you get them cloud ready"

I think we need to be a bit careful here, because it isn't always easy for us in current state to detect what stuff may or may not be missing to enable the application to run in a container. And this can vary across the different application types we support (ie. Appsody).